src/garrysmod/gamemodes/zombiesurvival/gamemode/howl/sv_howl.lua

```diff
@@ -0,0 +1,261 @@
+Howl = {}
```

Comment:
I thought about declaring a table within the `GAMEMODE` namespace, like `GAMEMODE.Admin`, but ultimately decided against it because I'm still not sure the direction we're going to take in building our fork out. 

For now, I'm leaving Howl in its own namespace because the functionality is sufficiently distinct from the core gameplay. In a perfect world, I'd put Howl in its own repository and reference it as a dependency.